### PR TITLE
Feature Virtual Reality Headset

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#5245

https://www.cyberbotics.com/doc/guide/the-user-interface?version=feature-open-vr#view-menu
https://www.cyberbotics.com/doc/guide/computer-peripherals?version=feature-open-vr

### CODE REVIEW

**Comment by DavidMansolino** [comment]
👍

**Comment by omichel** [comment]
:eyes:

File: guide/computer-peripherals.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Computer peripherals
+You may interface various computer peripherals with Webots in order to interact with the simulation.
+
+### Joystick
+You can use the [Joystick API](http://www.cyberbotics.com/reference/joystick) to get the state of the buttons and value of the axes in your controller. You may also (if your joystick supports it) use force feedback.
+You simply need to plug your peripherals and if needed install it's driver.
```
Comment (omichel) [inline guide/computer-peripherals.md]: it's &rarr; its

File: guide/the-user-interface.md [hunk 2]
```diff
@@ -143,6 +143,16 @@ original viewpoint.
 The `Fullscreen` item enables and disables displaying the 3D window on the
 entire screen.
 
+The `Virtual Reality Headset` submenu allows you to use a virtual reality headset such as HTC Vive or Oculus Rift to view the simulation.
+The `Enable` item allows you to switch the simulation view to the headset.
+The `Track headset position` and `Track headset orientation` items specifies if the headset position and orientation should be tracked and applied to the viewpoint.
+The `View left eye`, `View right eye` and `Empty view` radio buttons allows you to choose what should be displayed in the simulation view, you can either see the left eye image, right eye image or nothing.
+The headset installation procedure is descibed [here](computer-peripherals.md).
```
Comment (omichel) [inline guide/the-user-interface.md]: descibed &rarr; described

**Comment by DavidMansolino** [comment]
👍

File: guide/computer-peripherals.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+You simply need to plug your peripherals and if needed install its driver.
+A wide range of gamepad and steering wheel joysticks are supported.
+
+> **Note**:
+On Mac OS X, only the first 12 buttons and first 2 axes of the joystick are taken into account.
+
+### Virtual reality headset
+On Windows you can use a virtual reality headset to view the simulation.
+We use SteamVR to interface the headset with Webots, you will therefore need to install [Steam](http://store.steampowered.com/about/) and then [SteamVR](steam://run/250820).
+SteamVR is currently still unstable on Linux and Mac OS X, this is why we decided to support virtual reality headset only on Windows for now, but this might change in a near future.
```
Comment (tn12787) [inline guide/computer-peripherals.md]: in a near future -> in *the* near future

File: guide/the-user-interface.md [hunk 4]
```diff
@@ -143,6 +143,16 @@ original viewpoint.
 The `Fullscreen` item enables and disables displaying the 3D window on the
 entire screen.
 
+The `Virtual Reality Headset` submenu allows you to use a virtual reality headset such as HTC Vive or Oculus Rift to view the simulation.
```
Comment (tn12787) [inline guide/the-user-interface.md]: I would add a "the" before HTC Vive.

File: guide/the-user-interface.md [hunk 5]
```diff
@@ -143,6 +143,16 @@ original viewpoint.
...
+The `Enable` item allows you to switch the simulation view to the headset.
+The `Track headset position` and `Track headset orientation` items specifies if the headset position and orientation should be tracked and applied to the viewpoint.
+The `View left eye`, `View right eye` and `Empty view` radio buttons allows you to choose what should be displayed in the simulation view, you can either see the left eye image, right eye image or nothing.
```
Comment (tn12787) [inline guide/the-user-interface.md]: allows -> allow

Comment (tn12787) [inline guide/the-user-interface.md]: I'd also add a full stop after "simulation view".

**Comment by DavidMansolino** [comment]
👍

**Pull request merged**